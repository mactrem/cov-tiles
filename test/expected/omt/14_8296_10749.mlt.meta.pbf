²

water_name
id" 
geometry* 
class" 	
intermittent" 
name" 	
name:ar" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	<
housenumber
id" 
geometry* 
housenumber" 	F
	landcover
id" 
geometry* 
class" 	
subclass" 	2
landuse
id" 
geometry* 
class" 	é
place
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:el" 	
name:en" 	
name:fr" 	
name:int" 	
name:ja" 	

name:latin" 	
name:ru" 	
rank" 
name:ar" 	”
poi
id" 
geometry* 
agg:stop" 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" 
subclass" 	
layer" 
name:ru" 	
name:fr" 	
name:nonlatin" 	
indoor" 
level" 
name:uk" 	
name:ja" 	
name:pt" 	
name:ar" 	
name:zh" 	
name:ko" 	
name:it" 	F
water
id" 
geometry* 
class" 	
intermittent" ‰
transportation_name
id" 
geometry* 
brunnel" 	
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
subclass" 	
name:fr" 	
network" 	
ref" 	

ref:length" 
layer" V
building
id" 
geometry* 
render:height" 
render:min:height" ø
transportation
id" 
geometry* 
brunnel" 	
class" 	
layer" 
oneway" 
ramp" 
subclass" 	
foot" 	
bicycle" 	
surface" 	
horse" 	
service" 	
level" 
indoor" 