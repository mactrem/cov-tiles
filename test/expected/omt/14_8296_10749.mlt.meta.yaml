version: 1
featureTables {
  name: "water_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "housenumber"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "housenumber"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landcover"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landuse"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "place"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:el"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ja"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ru"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "poi"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "agg:stop"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name:ru"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:nonlatin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "indoor"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "level"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name:uk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ja"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:pt"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:zh"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ko"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:it"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "water"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "transportation_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:fr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "network"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref:length"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "building"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "render:height"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "render:min:height"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "transportation"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "oneway"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ramp"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "foot"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "bicycle"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "surface"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "horse"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "service"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "level"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "indoor"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
